charts/device-plugin-operator/README.md

```diff
@@ -44,11 +45,16 @@ You may also run `helm show values` on this chart's dependencies for additional
 |---------|-----------|
 | `manager.image.hub` | `intel` |
 | `manager.image.tag` | `` |
-| `kubeRbacProxy.image.hub` | `quay.io` |
-| `kubeRbacProxy.image.hubRepo` | `brancz` |
-| `kubeRbacProxy.image.tag` | `v0.18.1` |
-| `kubeRbacProxy.image.pullPolicy` | `IfNotPresent` |
+| `manager.devices` | `` |
 | `privateRegistry.registryUrl` | `` |
 | `privateRegistry.registryUser` | `` |
 | `privateRegistry.registrySecret` | `` |
-| `pullPolicy` | `IfNotPresent` |
\ No newline at end of file
+| `pullPolicy` | `IfNotPresent` |
+
+Defining `manager.devices` with a name-bool dictionary allows enabling only certain devices. The following will enable only fpga and gpu devices:
```

Comment:
instead of adding a new field, we could also document how `controllerExtraArgs` can be used for this?

Reply:
Sure. Not as easy for the end user, but doable. Should I change it?

Reply:
the proposed `devices` approach works for me